app/src/main/java/by/academy/jc/grinchik/ht2/Calculator.java

```diff
@@ -0,0 +1,7 @@
+package by.academy.jc.grinchik.ht2;
+
+public class Calculator {
+    public int getSum(int a, int b) {
```

Comment:
Please rename to just `sum()`. `Get` is used for special kind of methods called getters.